console/Command.php

```diff
@@ -57,6 +57,13 @@ class Command extends \lithium\core\Object {
 	public $plain = false;
 
 	/**
+	 * Shows text output with color or without
```

Comment:
Might be reworded as `Disables color output. Useful when piping command output into other commands.`. Also the text needs to be the same as in `Help::$noColor`